src/Playroom/Snippets/Snippets.tsx

```diff
@@ -94,15 +49,21 @@ export default ({ snippets, onHighlight, onClose }: Props) => {
     },
     50
   );
-  const debounceScrollToHighlighted = useDebouncedCallback(
-    scrollToHighlightedSnippet,
-    50
-  );
+
   const filteredSnippets = useMemo(
     () => filterSnippetsForTerm(snippets, searchTerm),
     [searchTerm, snippets]
   );
 
+  const highlightedItem =
+    typeof highlightedIndex === 'number'
+      ? document.getElementById(
+          `${filteredSnippets[highlightedIndex].group}_${filteredSnippets[highlightedIndex].name}_${highlightedIndex}`
```

Comment:
Given the formatting here, maybe its useful to have a function for generating the ID for a snippet? That way the formatting between here and the list item ID remain in sync and linked in code.